src/main/java/frc/robot/RobotContainer.java

```diff
@@ -51,9 +84,46 @@ public RobotContainer() {
 	 * joysticks}.
 	 */
 	private void configureBindings() {
-		swerve.setDefaultCommand(new DefaultDrive());
+		SWERVE.setDefaultCommand(new DefaultDrive());
 
-		driveController.a().onTrue(swerve.printOffsets());
-		driveController.x().onTrue(new InstantCommand(() -> imu.zeroYaw()));
+		InputManager.getInstance().init(
+				InputManager.Button.A_Button1, SWERVE.printOffsets(),
+				InputManager.Button.X_Button3, new InstantCommand(() -> PIGEON.zeroYaw()),
+				InputManager.Button.B_Button2, new InstantCommand(POSE_ESTIMATOR::resetOdometry));
+	}
+
+	public void periodic() {
+		POSE_ESTIMATOR.periodic();
+
+		xVelocity = PIGEON.getXVelocity();
```

Comment:
why not get these from the Pigeon when needed?

Reply:
That's there to update xVel, yVel, and rotationalVel later on if the sliders for them are used.

Reply:
I dont see any sliders, are they defined in another place?